diff --git a/dist/TypstDocument.js b/dist/TypstDocument.js
index eccaacf8c1323d25f613527e4fd6f68fd89f0096..c8df9bd6eee7a782e5ed5ef427fca0ce5037ece0 100644
--- a/dist/TypstDocument.js
+++ b/dist/TypstDocument.js
@@ -1,8 +1,8 @@
-import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
-import { useRef, useEffect } from 'react';
-import { withGlobalRenderer } from '@myriaddreamin/typst.ts/dist/esm/contrib/global-renderer.mjs';
 import * as typst from '@myriaddreamin/typst.ts';
-import htmlLayerCss from './typst.css?inline';
+import { withGlobalRenderer } from '@myriaddreamin/typst.ts/dist/esm/contrib/global-renderer.mjs';
+import { useEffect, useRef } from 'react';
+import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
+const htmlLayerCss = '';
 let moduleInitOptions = {
     beforeBuild: [],
     getModule: () => '/node_modules/@myriaddreamin/typst-ts-renderer/pkg/typst_ts_renderer_bg.wasm',
@@ -39,7 +39,7 @@ export const TypstDocument = ({ fill, artifact, format }) => {
         /// render after init
         withGlobalRenderer(typst.createTypstRenderer, moduleInitOptions, doRender);
     }, [displayDivRef, fill, artifact, format]);
-    return (_jsxs("div", { children: [_jsx("style", { children: htmlLayerCss }), _jsx("div", { className: "typst-app", style: { height: '0' }, ref: displayDivRef })] }));
+    return (_jsxs("div", { children: [_jsx("style", { children: htmlLayerCss }), _jsx("div", { className: "typst-app", style: { height: 'auto' }, ref: displayDivRef })] }));
 };
 /**
  * @deprecated since version v0.6.0
